### ISSUE TITLE
Github permissions

### ISSUE THREAD

**Issue by Andersos** [greenkeeperio/support#1]
Why do you need to be able to write user data?
![skjermbilde 2015-09-25 kl 15 54 48](https://cloud.githubusercontent.com/assets/288977/10102286/d7073270-639d-11e5-963e-4373066d0392.png)

**Comment by boennemann** [comment]
Hey @Andersos,

we tried to keep the scope as low as possible. If we want more data than your email address the only available scope is read _and write_.
I'll submit this for internal review – to see if we need anything other than the email, and then we can reduce the scope accordingly.

Thanks for your feedback and trying out Greenkeeper. I'll post the results of this as soon as we have them.

Best,
Stephan

**Comment by jdalton** [comment]
Ya, this stopped me in my tracks as well.

**Comment by jdalton** [comment]
Also I'm not a fan of opting for public and private by default. It should be public only by default then extra commands to go private.

**Comment by boennemann** [comment]
Hey @jdalton,

I'm really glad you found Greenkeeper and gave it a try.
You both are absolutely right, we shouldn't be doing it this way and be inconsiderate about scopes – in our own interest.

We'll review the user scope and probably reduce it down to user:email, and revert the default.

Thanks for your feedback – I'll let you know the minute it lands.

Best,
Stephan

**Comment by glenjamin** [comment]
:+1: for asking me about private modules later.

Would also like to be able to opt-in individual orgs instead of all-at-once.

**Comment by boennemann** [comment]
@jdalton @Andersos @glenjamin 

<img width="488" alt="screen shot 2015-09-26 at 16 27 18" src="https://cloud.githubusercontent.com/assets/908178/10118007/ffbc8764-646b-11e5-83d0-42b9e9955985.png">

The `user` scope got reduced down to `user:email`, which is effective immediately.

We still need to improve the UX around the `upgrade` command, so it's sending you to GitHub to increase the scope first, but once that's ready, the reversed default for public/private repos will land as well.

**Comment by Andersos** [comment]
Looks better now.

**Comment by boennemann** [comment]
I'm reopening this now, so we can close it once the reversed public/private default lands.

**Comment by glenjamin** [comment]
Should I raise a separate issue about individual org permissions, or should that be rolled into this as well?

**Comment by boennemann** [comment]
@Andersos @jdalton 

The new reversed default just landed as of https://github.com/greenkeeperio/greenkeeper/releases/tag/v2.0.0

Use `greenkeeper login --force` to downgrade existing accounts (unless you _want_ to use private).

@glenjamin GitHub already supports permissions for individual orgs on the authorization page.

Thanks for your patience and sorry for being inconsiderate to begin with.

Best,
Stephan
